src/gpl/src/nesterovPlace.cpp

```diff
@@ -830,6 +832,25 @@ int NesterovPlace::doNesterovPlace(int start_iter)
         log_->info(
             GPL, 89, "Routability end iteration: revert back to snapshot");
       }
+
+      if (!is_routability_need_) {
+        auto block = pbc_->db()->getChip()->getBlock();
+        int64_t end_routability_area = 0;
+        for (auto& nb : nbVec_) {
+          end_routability_area += nb->nesterovInstsArea();
+        }
+        double percent_diff
+            = 100.0 * (end_routability_area - original_area) / original_area;
+        log_->info(
```

Comment:
This should be a debugPrint as I don't think most users will know what it means (or care).